src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const log = [];
+  const clear = 'clear';
```

Comment:
```suggestion
  const CLEAR = 'clear';
```
Better write static variables in upper case